Gemfile

```diff
@@ -1,39 +1,39 @@
 source 'https://rubygems.org'
 
+gem 'responders', '~> 2.0'
 gem 'activeadmin'
-gem 'activerecord-import', '~> 0.2.9'
-gem 'acts-as-taggable-on', '~> 2.3.3'
-gem 'coffee-script-source', '1.1.2'
+gem 'activerecord-import', '0.4.1'
+gem 'acts-as-taggable-on', '3.5.0'
 gem 'colored', require: false
-gem 'devise', '~> 2.0.5'
+gem 'devise', '3.4.1'
 gem "foreman"
 gem 'geocoder', '~> 1.2'
 gem 'haml'
 gem 'honeybadger', '~> 2.0'
 gem 'mysql2', '~> 0.3.10'
 gem 'newrelic_rpm', '~> 3.14', '>= 3.14.0.305'
 gem 'oj'
-gem 'paperclip', '~> 2.8.0'
+gem 'paperclip', '4.1.1'
 gem 'progress', require: false
 gem 'pry-rails'
 gem 'rack-cors', require: 'rack/cors'
-gem 'rails', '3.2.22.5'
+gem 'rails', '4.2.11'
 gem 'rake', '10.0.3'
 gem 'rb-gsl', '1.16.0.6'
 gem 'rubyzip', '>= 1.0.0'
-gem 'sidekiq', '~> 4.0'
+gem 'sidekiq'
 gem 'sidekiq-unique-jobs', '3.0.12'
 gem 'sinatra', require: false
 gem 'thin', '~> 1.6', '>= 1.6.4'
 gem 'webpack-rails'
 gem 'test-unit', '~> 3.0'
+gem 'a9n'
+gem 'uglifier'
+gem 'sass-rails'
+gem 'coffee-rails'
 
-# Gems used only for assets and not required
-# in production environments by default.
-group :assets, :development do
-  gem 'coffee-rails', '~> 3.2.1'
-  gem 'sass-rails', '~> 3.2.2'
-  gem 'uglifier', '~> 1.2.7'
+group :development do
```

Comment:
Any reason we need two development groups? there is another one on line 40

Reply:
Good catch! Fixed it